stream-chat-android-client/src/main/java/io/getstream/chat/android/client/models/Config.kt

```diff
@@ -8,12 +8,14 @@ public data class Config(
     /**
      * Date of channel creation.
      */
+    @Deprecated("", level = DeprecationLevel.WARNING, replaceWith = ReplaceWith("createdAt"))
```

Comment:
```suggestion
    @Deprecated("Use createdAt instead", level = DeprecationLevel.WARNING, replaceWith = ReplaceWith("createdAt"))
```

This may seem redundant, but otherwise the annotation looks very awkward 😅

Reply:
Ok. I'll replace in the other places aswell